galeracluster/source/documentation/scriptable-sst.rst

```diff
@@ -103,6 +103,12 @@ These parameters are passed to all state transfer scripts, regardless of method
 
    ``--defaults-file`` The script is given the path to the ``my.cnf`` configuration file.
 
+   ``--total N`` This progress reporting parameter indicates the new SST stage and reports the estimated total work.
+
+   ``--complete N`` This progress reporting parameter reports the work completed so far.
+   
+   Progress reporting is also enabled for the ``clone`` SST method.
```

Comment:
This is a correct statement, but I'm not sure it belongs in the list of SST script parameters.